### PR TITLE
#367: Add 'go back to top' button

### PR DESCRIPTION
**Pull request by mariaogryczak**

### CODE REVIEW

File: src/main/webapp/app/modules/conflicts/shared/components/additional-details.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import '../shared-record-view.scss';
 import { connect } from 'react-redux';
+import ScrollToTop from 'react-scroll-up';
```
Comment (pmuchowski) [inline src/main/webapp/app/modules/conflicts/shared/components/additional-details.tsx]: did you added as a new dependency in package.json?

**Comment by pmuchowski** [comment]
@mariaogryczak remove the url conflict detection commit from this PR
When you creating a new branch you need to checkout to the develop first, otherwise all of the changes from the current branch will be moved to the new branch

File: src/main/webapp/app/modules/conflicts/shared/components/additional-details.tsx (hunk 1 above)
Comment (mariaogryczak) [reply src/main/webapp/app/modules/conflicts/shared/components/additional-details.tsx]: Before you wrote about it - no. Now I fixed it. 
I have one more question:
I also added an image of a button "content/images/backToTop_transparent_resized.png". Does it update automatically during commit or I have to add it in a different way?

File: src/main/webapp/app/modules/conflicts/shared/components/additional-details.tsx [hunk 2]
```diff
@@ -59,6 +60,11 @@ export class AdditionalDetails extends React.Component<IAdditionalDetailsProp, I
           ))}
           {additionalFields}
         </Collapse>
+        <div>
+          <ScrollToTop showUnder={150} duration={500} style={{ display: 'inline-block', marginRight: '10px' }}>
+            <FontAwesomeIcon size="3x" icon={'angle-up'} color="lightblue" />
+          </ScrollToTop>
```
Comment (pmuchowski) [inline src/main/webapp/app/modules/conflicts/shared/components/additional-details.tsx]: I think you can add the "Back to Top" text under the icon too

**Comment by pmuchowski** [comment]
@mariaogryczak there is still the "Improve URL conflict detection" commit in this PR, remove it

**Pull request merged**